x-pack/plugins/security_solution/public/detection_engine/rule_management/components/rule_details/translations.ts

```diff
@@ -98,13 +105,27 @@ export const BUILDING_BLOCK_FIELD_LABEL = i18n.translate(
   }
 );
 
-export const BUILDING_BLOCK_FIELD_DESCRIPTION = i18n.translate(
+export const BUILDING_BLOCK_ENABLED_FIELD_DESCRIPTION = i18n.translate(
   'xpack.securitySolution.detectionEngine.ruleDetails.buildingBlockFieldDescription',
   {
     defaultMessage: 'All generated alerts will be marked as "building block" alerts',
   }
 );
 
+export const BUILDING_BLOCK_DISABLED_FIELD_DESCRIPTION = i18n.translate(
+  'xpack.securitySolution.detectionEngine.ruleDetails.buildingBlockDisabledFieldDescription',
+  {
+    defaultMessage: 'Will not mark alerts as "building block" alerts',
```

Comment:
Ideally we should have a sentence without negation but we don't have a term `normal` alerts. It's better to consult with docs  folks regarding this text.

Reply:
Discussed this with Georgii. Decided to make add it as an item into the docs ticket.